dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -7835,7 +7833,7 @@ public Contentlet saveDraft(final Contentlet contentlet,
             throws IllegalArgumentException,DotDataException,DotSecurityException, DotContentletStateException, DotContentletValidationException {
 
         if (!InodeUtils.isSet(contentlet.getInode())) {
-            return checkin(contentlet, contentletRelationships, cats, permissions, user, false);
```

Comment:
I think there are more places to change on the saveDraft, where the respect front end roles is being hardcored

Reply:
done https://github.com/dotCMS/core/pull/18300/commits/1aa3444ac01b694697c3580a424d7e3566aabc35#diff-0e42bc8d57cb229663e56b48259b767dR7855-R7870